### PR TITLE
{bio} [goolf/1.7.20] coevol 

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
aw-4r14-n02 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/c51612ddaf85d2bb7989af668efc7329 for a full test report.

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
aw-4r14-n02 - Linux centos 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/db2ed1db28d4e057e366d411323ebb8d for a full test report.

File: easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+commit_id = '5d7fc52'
+version = '%s-02012018' % commit_id
+
+homepage = 'https://github.com/bayesiancook/coevol'
+description = 'Correlated evolution of substitution rates and quantitative traits'
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = ['https://github.com/bayesiancook/coevol/archive/']
+sources = ['%s.tar.gz' % commit_id]
```
Comment (boegel) [inline easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb]: please use the renaming support we have:

```python
sources = [{'download_filename': '%s.tar.gz' % commit_id, 'filename': SOURCE_TAR_GZ}]

File: easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+checksums = ['e2cb1227f452c4f781faf8ec60e483ca27ffd288380bcfa483886f445a775bb6']
+
+unpack_options = '--strip-components=1'
+
+skipsteps = ['configure', 'install']
+
+buildininstalldir = True
```
Comment (boegel) [inline easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb]: Why is this needed? Anything else other than the stuff in `bin/` required?

File: easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'ConfigureMake'
+
+name = 'coevol'
+commit_id = '5d7fc52'
+version = '%s-02012018' % commit_id
```
Comment (boegel) [inline easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb]: @pescobar Wouldn't it be better to simply use `20180201` as `version` (and don't mention the commit ID in the `version`), since that sorts better?

Comment (boegel) [reply easybuild/easyconfigs/c/coevol/coevol-5d7fc52-02012018-goolf-1.7.20.eb]: @pescobar please see #6642
